src/App.js

```diff
@@ -1,29 +1,20 @@
+import React from "react"
 import './App.css'
-
-//array to store list items
-const todoList = [
-  { id: 1, title: 'Complete Assignment' },
-  { id: 2, title: 'Complete Reading' },
-  { id: 3, title: 'Annotate notes' },
-  { id: 4, title: 'Watch Videos and update notes' }
-]
-
-//map function to create list items from todoList array
-let listItem = todoList.map(item => (
-    <li className="list--items" key={item.id}>
-      {item.title}
-    </li>
-  ))
+import TodoList from './TodoList'
+import AddTodoForm from './AddTodoForm'
 
 
 function App() {
 
+  //state variable and set function to set state to value added by user input
+  const [newTodo, setNewTodo] = React.useState({value:""});
```

Comment:
```suggestion
  const [newTodo, setNewTodo] = React.useState("");
```

I make life easier, I would use state as a string instead of an object. It will make it easier when dealing with the todo list array in later lesson.